views/index.html

```diff
@@ -6,14 +6,14 @@
       <title>Timestamp Microservice</title>
       <link rel="shortcut icon" href="https://cdn.hyperdev.com/us-east-1%3A52a203ff-088b-420f-81be-45bf559d01b1%2Ffavicon.ico" type="image/x-icon"/>
       <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
-      <link href="style.css" rel="stylesheet" type="text/css">
+      <link href="/public/style.css" rel="stylesheet" type="text/css">
```

Comment:
```suggestion
      <link href="style.css" rel="stylesheet" type="text/css">
```
It seems Glitch puts everything in the `/public` dir into the `/` route.

Reply:
I tried this very quickly on my remix of this project and none of the styling was applied when I opened the project in another window. Maybe it's the way we're serving the `/public` directory in `server.js`?:

```js
app.use('/public', express.static(process.cwd() + '/public'));
```

Reply:
My bad, I thought that was Glitch specific thing!

Where's 
```js
app.use('/public', express.static(process.cwd() + '/public'));
```
coming from?  I can't find that anywhere and the PR has this:
https://github.com/freeCodeCamp/boilerplate-project-timestamp/blob/2e7849ad37716fbffdaacc61e60716dfc69584b2/server.js#L14

Which does what I thought Glitch was doing, namely mount everything inside `/public` at the root of the site.

Here's a project which I started by cloning this PR and then added `href="style.css"`: https://glitch.com/~succinct-clover-aardwolf.

Reply:
@ojeytonwilliams, no problem at all! I'm still trying to wrap my head around all of this myself. Will be nice to get off Glitch entirely so there's not that extra layer of pastel colored complexity.

Thank you for pointing this out -- it's yet another place where the solution and boilerplate have somehow merged over the years. (Insert copy of your profile picture).

Mounting everything in the `/public` directory to the root is much easier and cleaner. I'll go in and update the PR now.